### DISCUSSION TITLE
ödev

### DISCUSSION THREAD

**Discussion by ozanacarr** [gusanmaz/bmb212-algorithms-2024#7] [General]
hocam 10 sorunun hepsi mi sarı başlıklardan olmak zorunda yoksa sarı başlıklardan en az 2 tane örnek yeterli mi geri kalanları işaretlenmemiş başlıklardan yapabiliyor muyuz?

**Comment by ozanacarr** [comment]
sarı başlıkları fark etmeden diğer başlıklardan da yapmıştım

**Comment by gusanmaz** [comment]
Çözdüğün soruların başlıklarını ve o başlıkta çözülen soru sayısını yazabilir misin?

Sarı başlıklar dersin temel konuları aralarında olduğu ve derste bu konular işlenecek konular içinde olduğu için işaretlendi.

**Reply by ozanacarr** [reply]
hashmap ,1D DP ,sliding window,two pointers her başlık için ikişer  soru. bu başlıklar için çözümeler kabul edilir mi? yoksa yeniden mi yapmam gerek

**Reply by gusanmaz** [reply]
Özellikle Dinamik Programlama dersin kapsamının dışında ama bu konuyu biliyorsun soruları kodladığına göre.

Uygun benim için. Yine de istersen diğer başlıklardan biraz soru kodlayıp 10 üzeri soru için kod gönderebilirsin.
